backend/typescript/services/implementations/campService.ts

```diff
@@ -1291,8 +1330,9 @@ class CampService implements ICampService {
       const stripeDropoffProduct = await createStripeDropoffProduct(camp.name);
       const stripePickupProduct = await createStripePickupProduct(camp.name);
 
+      // Create the dropoff
       let priceIds = { dropoffPriceId: "", pickupPriceId: "" };
-      if (camp.active) {
+      if (camp.active && camp.dropoffFee && camp.pickupFee) {
```

Comment:
Same here as above - is it guaranteed that both fees are paid together? I assumed a parent can opt in for just one

Reply:
Actually I see that EDLP opt in is all in one, so perhaps disregard the above.